invenio_rdm_records/records/api.py

```diff
@@ -35,7 +36,9 @@ class CommonFieldsMixin:
         extensions=[
             EDTFDumperExt('metadata.publication_date'),
             EDTFListDumperExt("metadata.dates", "date"),
+            RecordStatusDumperExt("metadata.status"),
```

Comment:
I think these properties should not be inside the ``metadata`` key but rather as top-level keys.